handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslContext.java

```diff
@@ -184,14 +184,17 @@ protected void deallocate() {
                                    int mode, Certificate[] keyCertChain, ClientAuth clientAuth, String[] protocols,
                                    boolean startTls, boolean enableOcsp, boolean leakDetection) throws SSLException {
         this(ciphers, cipherFilter, toNegotiator(apnCfg), sessionCacheSize, sessionTimeout, mode, keyCertChain,
-                clientAuth, protocols, startTls, enableOcsp, leakDetection);
+                clientAuth, protocols, startTls, enableOcsp, leakDetection, new Map.Entry[0]);
```

Comment:
the constructor we are calling does a `ctxOptions != null` check ... can we just pass `null` here? If now does it make sense to cache this empty array?

Reply:
actually we don't need the constructor anymore... Removed it